### ISSUE TITLE
Ipywidget bokeh messes up jupyter lab css

### ISSUE THREAD

**Issue by govinda18** [bokeh/ipywidgets_bokeh#65]
Ipywidget bokeh does not seem to work well with jupyter lab's css. Whenever it is loaded on the page with lab's css, it messes up the css of the lab.  Here is a simple reproducer:

```python
import panel as pn
import ipywidgets_bokeh
import ipywidgets as w
pn.extension()
pn.Row(ipywidgets_bokeh.IPyWidget(widget=w.HTML("asd")))
```

![image](https://user-images.githubusercontent.com/25079291/199188643-be9a5e59-8df0-4292-8d47-78ef696017b8.png)

Note that the above is just for a reproducer. My use case requires me to load some javascript extensions I need dynamically on the page. For now, a workaround for me is to to not load it when I am in an ipynb environment.

**Comment by philippjfr** [comment]
In Panel docs we warn about this and ask you to load ipywidgets via the extension (i.e. with `pn.extension('ipywidgets')`) so that we can appropriately dispatch to the appropriate renderer depending on the environment we are running in.

**Comment by govinda18** [comment]
> In Panel docs we warn about this and ask you to load ipywidgets via the extension (i.e. with `pn.extension('ipywidgets')`) so that we can appropriately dispatch to the appropriate renderer depending on the environment we are running in.

Yep, I remember seeing that. My issue was actually related to https://github.com/holoviz/panel/issues/4074. With the workaround I mentioned in the issue, ipywidgets-bokeh will always load on the page

**Comment by ndmlny-qs** [comment]
@govinda18 I am going to work on #67 to see if updating `ipywidgets_bokeh` to use Bokeh 3.0 fixes these CSS issues. If it doesn't, then I'll try to figure out the culprit and will update this issue with anything I find out.
